### PR TITLE
fix(PDFReports): RHICOMPL-2639 - Define OS version

### PR DESCRIPTION
**Pull request by LightOfHeaven1994**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: src/SmartComponents/ReportDownload/Components/ReportPDF.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const ReportPDF = ({ data, ssgFinder }) => {
             // TODO: correct left side styling
             rows={[
               ['Policy type', policy.policyType],
-              ['Operating system', `RHEL ${policy.osMajorVersion}`],
+              ['Operating system', `RHEL ${policy.majorOsVersion}`],
```
Comment (skateman) [inline src/SmartComponents/ReportDownload/Components/ReportPDF.js:71]: WAT? Is this conventional? :thinking:

Comment (skateman) [reply src/SmartComponents/ReportDownload/Components/ReportPDF.js:71]: Okay, there's both `majorOsVersion` and `osMajorVersion` in the GQL schema. We should stick with the `osMajorVersion` instead, but let's get this in so it's fixed and I'll refactor that later...
i1-will-refactor-that-code-later-today-and-other-31878628

**Review by skateman — APPROVED** [review]
The Seal of Approval

File: src/SmartComponents/ReportDownload/Components/ReportPDF.js (hunk 1 above)
Comment (LightOfHeaven1994) [reply src/SmartComponents/ReportDownload/Components/ReportPDF.js:71]: Agree, thanks

**Pull request merged**